src/master-playlist-controller.js

```diff
@@ -1224,6 +1222,25 @@ export class MasterPlaylistController extends videojs.EventTarget {
       }
     }
 
+    // fmp4 relies on browser support, while ts relies on muxer support
```

Comment:
Originally I was doing this by playlist but after looking at the tests and testing more videos it was entirely inadequate. We have to check for support when we get all `trackinfo` just before we try to create source buffers.